bundles/org.openhab.binding.yioremote/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="yioremote"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="yioRemoteDock">
+		<label>YIO Remote Dock</label>
+		<description>YIOremote Dock Binding Thing</description>
+		<channel-groups>
+			<channel-group id="input" typeId="input"/>
+			<channel-group id="output" typeId="output"/>
+		</channel-groups>
+		<config-description>
+			<parameter name="host" type="text" required="true">
+				<label>Network address of the YIO Remote Dock</label>
```

Comment:
Labels should be all upper case. The binding name could be removed, as it is redundant.
```suggestion
				<label>Network Address</label>
```

Reply:
solved